pkg/operator/targetconfigcontroller/targetconfigcontroller.go

```diff
@@ -341,6 +358,115 @@ func manageKubeAPIServerCABundle(lister corev1listers.ConfigMapLister, client co
 	return resourceapply.ApplyConfigMap(client, recorder, requiredConfigMap)
 }
 
+func managePodNetworkConnectivityChecks(ctx context.Context, client kubernetes.Interface,
```

Comment:
Can we find a way to isolate this in another control loop?

Reply:
Done.